changelog/4.0.0.md

```diff
@@ -0,0 +1,68 @@
+* Exposed `github.com/RackSec/srslog` to anko scripts, allowing scripts to send syslog messages
+* Exposed `encoding/base64`, `encoding/xml`, and `encoding/hex` packages to anko scripts
+* Added duration filtering and extraction for netflow and ipfix modules
+* Improved autoextractor behavior in multi-tag queries with disjoint field sets
+* Improved webserver→indexer ingest functionality (entries are now ingested in parallel rather than blocking the control port)
+* Improved recovery for corrupted shards & indexes
+* Added `Max-Log-Files` configuration option to limit the number of log files in `/opt/gravwell/log`
+* `stats` module now properly handles TAG, e.g. `tag=* stats unique_count(TAG) as foo | gauge foo`
+* Improved handling of search deletion (admin users were unable to delete other users' ACTIVE/SAVED searches)
+* `syslog` module will no longer drop entries which it fails to parse as syslog entries (matching the behavior of other extraction modules)
+* Fixed bug where entries could come back with entry DATA fields during a replication hot-failover scenario
+* Fixed bug where a query containing a macro immediately followed by a newline would fail to parse
+* Fixed error where an improperly shut-down indexer could cause replication failures
+* Fixed issue where an error in the renderer module would be marked as originating from a different module instead
+* Fixed incorrect time reversal behavior of sort module in certain cases
+* Fixed off-by-one error in query tag handling which could cause crashes
+* Fixed bug in macro creation in which groups were not properly preserved
+* Fixed SSO bug where user removal from a group in Active Directory was not mirrored in Gravwell
+* Fixed incorrect nil pointer handling in scheduled search `httpPost` function
+* Fixed potential webserver crash during shutdown due to lingering websockets
+* Fixed bug where `gauge` module could not parse multiple `(magnitude label)` specifications
+* Fixed bug in `gauge` module where enumerated values containing underscores were not parsed correctly
+
+## Ingesters & Ingest Library Changes
+* Implemented new, more efficient cache system
+* Added preprocessor to handle Amazon VPC flow logs
+* Fixed bug where cached entries could fail tag translation on ingester restart
+* Kinesis ingester now reports metrics via `gravwell` tag
+* Kafka Federator now supports TLS-encrypted Kafka transports
+* Fixed line number reporting on ingester log messages
+* Improved logging in early ingester startup
+* Added code to handle Zeek's tendency to leak PIDs
+
+## Kits
+* Network Enrichment: Provides databases for enriching network data (geoip, ASN, IP protocols, etc.)
+* CoreDNS: Analyze entries from the [Gravwell CoreDNS plugin](https://github.com/gravwell/coredns)
+* Netflow v5: Analyze Netflow v5 flow records
+* IPFIX: Analyze IPFIX flow records
+* Grok: Pre-built database of extraction patterns for use with the [grok module](https://docs.gravwell.io/docs/#!search/grok/grok.md)
+
```

Comment:
go ahead and call out the following additional kits:
* zeek
* Windows
* syslog

They will be ready for launch or shortly thereafter